### PR TITLE
Align popIBM.R with diseasy documentation standard - part 2

### PR DESCRIPTION
**Pull request by RasmusSkytte**
Extending on #5 , this PR tracks the changes to manuscripts/popIBM/popIBM.R in terms of documentation standard

### CODE REVIEW

File: manuscripts/popIBM/popIBM.R [hunk 1]
```diff
@@ -480,10 +512,10 @@ sim_list <- foreach(run_this = (first_run - 1 + 1:n_runs), .packages = "data.tab
...
         on = "municipality_id",
-        kom_fac := lockdown_municipality_fac
+        municipality_fac := lockdown_municipality_fac
       ]
 
-      population[, lockdown_max := pmax(parish_fac, kom_fac)]
+      population[, lockdown_max := pmax(parish_fac, municipality_fac)]
 
       # Weighted sum as lockdown factor
       population[, lockdown_fac := 1 * (1 - lockdown_max) + lockdown_factor * lockdown_max]
```
Comment (RasmusSkytte) [inline manuscripts/popIBM/popIBM.R:527]: This computation needs some context comments

File: manuscripts/popIBM/popIBM.R [hunk 2]
```diff
@@ -318,6 +324,10 @@ sim_list <- foreach(run_this = (first_run - 1 + 1:n_runs), .packages = "data.tab
 
         # Group population by age group and vaccinated / un-vaccinated
         tmp <- ibm[, .N, keyby = .(age_groups, !(vac_time < breaks_vac[[1]] | is.na(vac_time)))]
+
+        # Then merge population counts with testing behavior in age and vaccine groups (n_test_age_groups_int_vac).
+        # This gives the ???
+        # (n_test_age_groups_int_vac is loaded as part of init)
         t_pop_age_vac <- tmp[n_test_age_groups_int_vac, , on = c("age_groups", "vac_time")]
         t_pop_age_vac[is.na(N), N := 0]
```
Comment (RasmusSkytte) [inline manuscripts/popIBM/popIBM.R:339]: This part in general needs additional context

File: manuscripts/popIBM/popIBM.R [hunk 3]
```diff
@@ -494,17 +526,17 @@ sim_list <- foreach(run_this = (first_run - 1 + 1:n_runs), .packages = "data.tab
...
         .(inf_persons = sum(lockdown_fac * non_isolated * vac_fac_trans)),
         by = .(municipality_id, age_groups)
       ][all_pop_combi, on = c("municipality_id", "age_groups")]
 
       inf_persons_municipality[is.na(inf_persons), inf_persons := 0]
-      inf_persons_municipality[, inf_persons := inf_persons * v_rel_beta[k]]
+      inf_persons_municipality[, inf_persons := inf_persons * v_rel_beta[variant_id]]
 
       inf_pressure <- inf_persons_municipality[
         , current_beta %*% inf_persons, by = .(municipality_id)
```
Comment (RasmusSkytte) [inline manuscripts/popIBM/popIBM.R:548]: This part in general needs additional context

File: manuscripts/popIBM/popIBM.R [hunk 4]
```diff
@@ -556,10 +588,10 @@ sim_list <- foreach(run_this = (first_run - 1 + 1:n_runs), .packages = "data.tab
...
     # Implement the effect of vaccination
-    for (k in 1:n__vac){
+    for (vac_id in 1:n_vac){
 
-      ibm[vac_type == k & vac_time == v_vac_tt_effect[k],
-          `:=`(vacF_ec = delta_vac_effect[k],
+      ibm[vac_type == vac_id & vac_time == v_vac_tt_effect[vac_id],
+          `:=`(vacF_ec = delta_vac_effect[vac_id],
                prob_hospital = delta_red_hospital_risk * prob_hospital,
                vac_fac_trans = red_transmission_vac)]
```
Comment (RasmusSkytte) [inline manuscripts/popIBM/popIBM.R:602]: This I don't quite understand. 
Getting a vaccine changes a bunch of parameters for the individual. But these are all related to Delta variant it seems. What about infection with Alpha / Original?

Comment (kaare-gr) [reply manuscripts/popIBM/popIBM.R:604]: Wrong. Code I have replaced with correct.

File: manuscripts/popIBM/popIBM.R (hunk 1 above)
Comment (kaare-gr) [reply manuscripts/popIBM/popIBM.R:529]: done

File: manuscripts/popIBM/popIBM.R (hunk 2 above)
Comment (kaare-gr) [reply manuscripts/popIBM/popIBM.R:341]: done. It is a merger to make zero groups.

**Pull request merged**